test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -90,11 +97,57 @@ public void AutoCompleteFindCompletions()
             var completions = autoCompleteService.GetCompletionItems(
                 result.TextDocumentPosition,
                 result.ScriptFile,
-                result.ConnectionInfo);
+                result.ConnectionInfo).Result;
 
             Assert.True(completions.Length > 0);
         }
 
+        public static string AssemblyDirectory
+        {
+            get
+            {
+                string codeBase = Assembly.GetExecutingAssembly().CodeBase;
+                UriBuilder uri = new UriBuilder(codeBase);
+                string path = Uri.UnescapeDataString(uri.Path);
+                return Path.GetDirectoryName(path);
+            }
+        }
+
+        /// <summary>
+        /// This test tests completion extension interface
```

Comment:
>This test tests completion extension interface [](start = 12, length = 46)

Generally I'd prefer these be more descriptive - this tells us nothing useful. What part of the interface is it testing? What is it doing to verify that? Given the name and description I have no idea what this is doing without reading the code itself. Adding comments to the code itself to help this (so the summary doesn't get too verbose) is useful too.

Reply:
Added more details to the description.